identity-and-access-management/secret-manager/concepts.mdx

```diff
@@ -40,6 +40,16 @@ A PEM or **Privacy Enhanced Mail** file is a security certificate file used to e
 
 </Concept>
 
+<Concept>
+
+## Region
+
+A region is the geographical area in which your folders, secrets, and versions are stored and run. They are duplicated on all [Availability Zones](/compute/instances/concepts/#availability-zone) of the selected region (Paris, Amsterdam or Warsaw).
+
+Secret Manager allows you to filter your resources by region. This allows you to better sort through your resources, if you have created them in different regions but with the same name, for example.
```

Comment:
I think we should say select more than filter, because filter implies there is a global view, that does not exist.